docs/02-app/02-api-reference/04-functions/unstable_cache.mdx

```diff
@@ -39,10 +39,36 @@ const data = unstable_cache(fetchData, keyParts, options)()
   - `tags`: An array of tags that can be used to control cache invalidation.
   - `revalidate`: The number of seconds after which the cache should be revalidated. Omit or pass `false` to cache indefinitely or until matching `revalidateTag()` or `revalidatePath()` methods are called.
 
+> **Tags vs KeyParts**:
+>
+> - tags: They are used for cache invalidation, NextJS does not use it to identify the function signature. 
```

Comment:
I am not sure this adds more clarity than the bullets above. Can we modify there if needed?

Reply:
Good idea, the description of keyparts above is pretty convoluted, I can simplify it.